mathics/builtin/__init__.py

```diff
@@ -216,3 +216,12 @@ def is_builtin(var):
 _builtins = {}
 
 add_builtins(new_builtins)
+
+
+display_operators_set = set()
```

Comment:
I didn't find where it is used.

Reply:
This is used in `_BoxedString`

Reply:
In ``mathics.core.atom.String.boxes_to_mathml`` this set is built each time the method is called. I just move it to ``mathics.builtin.base`` to do it just once.